### PR TITLE
Add Apache Flink release 1.8.1

### PR DESCRIPTION
**Pull request by sunjincheng121**
Adds release announcement and download(English and Chinese) link for 1.8.1.
Dates on announcements are still TBD and should be updated accordingly once release happens.

### CODE REVIEW

File: _posts/2019-06-16-release-1.8.1.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+date:   2019-06-16 12:00:00
+categories: news
+authors:
+- till:
+  name: "Jincheng Sun"
+---
+
+The Apache Flink community released the first bugfix version of the Apache Flink 1.8 series.
+
+This release includes more than 40 fixes and minor improvements for Flink 1.8.1, The list below includes a detailed list of all fixes.
```
Comment (MarkSfik) [inline _posts/2019-06-16-release-1.8.1.md]: This release includes more than 40 fixes and minor improvements for Flink 1.8.1.  Below we include a detailed list of all resolved issues, sub-tasks, bug fixes and improvements.

Comment (MarkSfik) [inline _posts/2019-06-16-release-1.8.1.md]: I would update this with: 

This release includes more than 40 fixes and minor improvements for Flink 1.8.1. Below we include a detailed list of all improvements, sub-task and bug fixes in the latest release.

Comment (sunjincheng121) [reply _posts/2019-06-16-release-1.8.1.md]: I like add `all improvements, sub-tasks and bug fixes`. :)

File: _config.yml [hunk 2]
```diff
@@ -55,23 +55,23 @@ flink_releases:
   -
       version_short: 1.8
       binary_release:
-          name: "Apache Flink 1.8.0"
+          name: "Apache Flink 1.8.1"
           scala_211:
               id: "180-download_211"
```
Comment (hequn8128) [inline _config.yml]: 180 to 181. Same for other places.

File: _posts/2019-06-16-release-1.8.1.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title:  "Apache Flink 1.8.1 Released"
+date:   2019-06-16 12:00:00
+categories: news
+authors:
+- till:
```
Comment (wuchong) [inline _posts/2019-06-16-release-1.8.1.md]: jingcheng?

File: _config.yml (hunk 2 above)
Comment (wuchong) [reply _config.yml]: +1

**Comment by sunjincheng121** [comment]
Thanks for the review @hequn8128 @wuchong 
I have updated the PR accordingly :)

**Comment by tzulitai** [comment]
+1, looks good to merge minus the date update.

**Comment by sunjincheng121** [comment]
Thanks @tzulitai  @hequn8128 @wuchong  :) Merging....

**Pull request closed without merging**